convert2rhel/actions/handle_packages.py

```diff
@@ -0,0 +1,104 @@
+                " for the following third party packages:\n" % system_info.name
+            )
+            pkghandler.print_pkg_info(third_party_pkgs)
+            utils.ask_to_continue()
+        else:
+            logger.info("No third party packages installed.")
+
+
+class RemoveExcludedPackages(actions.Action):
+    id = "REMOVE_EXCLUDED_PACKAGES"
+
+    @actions._action_defaults_to_success
+    def run(self):
+        """
+        Certain packages need to be removed before the system conversion,
+        depending on the system to be converted.
+        """
+        logger.task("Convert: Remove excluded packages")
+        logger.info("Searching for the following excluded packages:\n")
+        try:
+            pkghandler.remove_pkgs_with_confirm(system_info.excluded_pkgs)
+            # TODO: Handling SystemExit here as way to speedup exception
+            # handling and not refactor contents of the underlying function.
+        except SystemExit as e:
+            # TODO(r0x0d): Places where we raise SystemExit and need to be
+            # changed to something more specific.
+            #   - When we can't remove a package.
+            self.set_result(status="ERROR", error_id="PACKAGE_REMOVAL_FAILED", message=str(e))
+            return
+
+
+class RemoveRepositoryFilesPackages(actions.Action):
+    id = "REMOVE_REPOSITORY_FILES_PACKAGES"
+
+    @actions._action_defaults_to_success
+    def run(self):
+        """
+        Remove those non-RHEL packages that contain YUM/DNF repofiles
+        (/etc/yum.repos.d/*.repo) or affect variables in the repofiles (e.g.
+        $releasever).
+
+        Red Hat cannot automatically remove these non-RHEL packages with other
+        excluded packages. While other excluded packages must be removed before
+        installing subscription-manager to prevent package conflicts, these
+        non-RHEL packages must be present on the system during
+        subscription-manager installation so that the system can access and
+        install subscription-manager dependencies. As a result, these non-RHEL
+        packages must be manually removed after subscription-manager
+        installation.
+        """
+        logger.task("Convert: Remove packages containing .repo files")
+        logger.info("Searching for packages containing .repo files or affecting variables in the .repo files:\n")
+        try:
+            pkghandler.remove_pkgs_with_confirm(system_info.repofile_pkgs)
+            # TODO: Handling SystemExit here as way to speedup exception
+            # handling and not refactor contents of the underlying function.
+        except SystemExit as e:
+            # TODO(r0x0d): Places where we raise SystemExit and need to be
+            # changed to something more specific.
+            #   - When we can't remove a package.
+            self.set_result(status="ERROR", error_id="PACKAGE_REMOVAL_FAILED", message=str(e))
+            return
```

Comment:
This return can be removed since it's at the end of the file right?

Reply:
```suggestion
```